app/src/main/java/com/programmers/library/util/ScannerUtil.java

```diff
@@ -0,0 +1,24 @@
+package com.programmers.library.util;
+
+import java.util.Scanner;
+
+public class ScannerUtil {
+
```

Comment:
```suggestion
    private ScannerUtil() {}
```

유틸 클래스를 생성하여 관리할 때는 클래스를 직접 생성하지 못하도록 막아줘야 합니다.